README.md

```diff
@@ -59,16 +59,17 @@ Getting started
 For iCE40 support, install [Project IceStorm](https://github.com/YosysHQ/icestorm) to `/usr/local` or another location, which should be passed as `-DICESTORM_INSTALL_PREFIX=/usr` to CMake. Then build and install `nextpnr-ice40` using the following commands:
 
 ```
-cmake . -DARCH=ice40
-make -j$(nproc)
-sudo make install
+cmake . -B build -DARCH=ice40
```

Comment:
I think just doing a mkdir/cd build beforehand would be clearer than the -B/-Cs myself, not sure if you have a strong preference here

Reply:
No preference to be honest. There are always multiple ways, whatever looks better to others is fine by me